sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_connection.py

```diff
@@ -422,7 +436,7 @@ def _incoming_open(self, channel, frame):
             _LOGGER.error("OPEN frame received in the OPENED state.", extra=self._network_trace_params)
             self.close()
         if frame[4]:
-            self._remote_idle_timeout = frame[4] / 1000  # Convert to seconds
+            self._remote_idle_timeout = cast(float, frame[4] / 1000)  # Convert to seconds
```

Comment:
it would, but I don't want to change the type to be a float but rather to be treated as a float (which could be an int from a mypy pov ).  mypy was complaining about the line below this, where multiplication was happening on a variable that is defined as `Optional[float]`.